### ISSUE TITLE
Building nextcloud FULL image fails on "creating libtool"

### ISSUE THREAD

**Issue by bolet** [nextcloud/docker#1551]
I'm using the contents of this folder: https://github.com/nextcloud/docker/tree/master/.examples/dockerfiles/full/fpm-alpine

When building the image with `docker build -t nextcloud-full:fpm-alpine .` I get the following:
```
...
creating libtool
appending configuration tag "CXX" to libtool
configure: patching config.h.in
configure: creating ./config.status
config.status: creating config.h
make: /bin/sh: Operation not permitted
make: *** [Makefile:192: bz2.lo] Error 127
...
```

This used to work for many releases till a few months ago.

In case it's useful: I'm running on a debian buster armv7 platform.

**Comment by J0WI** [comment]
Duplicate of https://github.com/docker-library/php/issues/1177

TL;DR: upgrade do Docker 20.10.7+

**Comment by bolet** [comment]
The docker apt repo only provides 19.03.13 for debian buster armv7. Before looking for alternate ways to install a more recent docker version on the target server, I tried cross-building from my desktop amd64 (ubuntu docker 20.10.2), and it worked. For now I'll keep it that way. Thanks a lot for your help @J0WI !
